### ISSUE TITLE
Handle `num(wei/m)` argument from viper's `proc_reward`

### ISSUE THREAD

**Issue by djrtwo** [RigoBlock/SolidityCasper#2]
Viper sometimes uses "unit"s for types such as `num(wei/m)`. Documentation on this feature is scarce. Figure out how units are represented at a byte level and decide what solidity type can represent `num(wei/m)` for the `proc_reward` method.

**Comment by gabririgo** [comment]
yes we need to understand this better. For now I have been able to fix the bytes issue as it has to be written as follows:
bytes in viper are bytes32, (bytes in solidity default to bytes 32 as well)
for arrays of data with maximum lenght: bytes[1024] and bytes[1000]

**Comment by djrtwo** [comment]
arguments of the form bytes[N] are not going to give you what you want (and I don't think will even compile). bytes is already an array of bytes (unbounded in length) so bytes[1024] would be an array of 1024 of bytes arrays.
